client/allocdir/fs_linux.go

```diff
@@ -30,8 +30,15 @@ func linkDir(src, dst string) error {
 
 // unlinkDir unmounts a bind mounted directory as Linux doesn't support
 // hardlinking directories.
+//
```

Comment:
Nit pick but can you just add that to the existing comment. No need for extra line